web/packages/teleport/src/TopBar/TopBar.tsx

```diff
@@ -172,10 +172,129 @@ export function TopBar() {
 }
 
 export const TopBarContainer = styled(TopNav)`
-  height: 72px;
-  background-color: inherit;
-  padding-left: ${p => `${p.theme.space[p.navigationHidden ? 2 : 6]}px`};
+  background: ${p => p.theme.colors.levels.surface};
   overflow-y: initial;
+  overflow-x: none;
   flex-shrink: 0;
+  z-index: 1000;
   border-bottom: 1px solid ${({ theme }) => theme.colors.spotBackground[0]};
+
+  height: 48px;
+  @media screen and (min-width: ${p => p.theme.breakpoints.small}px) {
+    height: 56px;
+  }
+  @media screen and (min-width: ${p => p.theme.breakpoints.large}px) {
+    height: 72px;
+  }
+
+  box-shadow: 0px 1px 3px 0px rgba(0, 0, 0, 0.12),
+    0px 1px 1px 0px rgba(0, 0, 0, 0.14), 0px 2px 1px -1px rgba(0, 0, 0, 0.2);
 `;
+
+const TeleportLogo = ({ CustomLogo }: TopBarProps) => {
+  const theme = useTheme();
+
+  return (
+    <HoverTooltip
+      anchorOrigin={{ vertical: 'bottom', horizontal: 'center' }}
+      transformOrigin={{ vertical: 'top', horizontal: 'center' }}
+      tipContent="Teleport Resources Home"
+    >
+      <Link
+        css={`
+          cursor: pointer;
+          height: 100%;
+          display: flex;
+          width: 190px;
+          @media screen and (min-width: ${p => p.theme.breakpoints.medium}px) {
+            width: 256px;
+          }
+
+          transition: background-color 0.1s linear;
+          &:hover {
+            background-color: ${p =>
+              p.theme.colors.interactive.tonal.primary[0]};
+          }
+          align-items: center;
+        `}
+        to="/web"
```

Comment:
define in config as a route?

Reply:
Thanks. I guess it was already defined under `cfg.routes.root` so i'll use that!